recipes/eleuther_eval.py

```diff
@@ -89,17 +89,28 @@ def tok_encode(self, text: str, **kwargs) -> List[int]:
         # https://github.com/pytorch-labs/gpt-fast/blob/main/eval.py#L123.
         return self._tokenizer.encode(text=text, add_bos=False, add_eos=False)
 
-    def tok_decode(self, tokens: List[int], **kwargs) -> str:
+    def tok_decode(self, tokens: Union[int, List[int]], **kwargs) -> str:
+        if isinstance(tokens, int):
+            tokens = [tokens]
         return self._tokenizer.decode(tokens)
 
     def _model_call(self, inps: torch.Tensor, **kwargs) -> torch.Tensor:
         return self._model(inps)
 
     def _model_generate(self, *args, **kwargs):
```

Comment:
Thanks for this awesome PR! It seems that we also need to implement 'tok_batch_encode' method?